### PR TITLE
GNOME Updates (Mainly 41.0 → 41.1)

### PR DESCRIPTION
**Pull request by fabianhjr**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/tracker/default.nix [hunk 1]
```diff
@@ -31,27 +31,20 @@
...
   };
 
   patches = [
     (substituteAll {
       src = ./fix-paths.patch;
       inherit asciidoc;
     })
-  ] ++ lib.optionals (stdenv.hostPlatform.isi686) [
-    # Upstream: https://gitlab.gnome.org/GNOME/tracker/-/issues/332
-    (fetchpatch {
```
Comment (fabianhjr) [inline pkgs/development/libraries/tracker/default.nix:50]: Should also remove fetchpatch from inputs.

**Comment by jtojnar** [comment]
Thanks, changes look good. Though VTE upgrade reportedly breaks aarch64 so it was reverted https://github.com/NixOS/nixpkgs/pull/142941. It might be also useful to merge the stuff that does not cause mass rebuilds to master so that it can still make it into 21.11 (or we might need someone to backport it).

**Comment by jtojnar** [comment]
Also gnome-shell also needs the color picker patch removed.

**Comment by fabianhjr** [comment]
Apologies, I messed up the order of the retargetting to master.

Reduced the PR to those that shouldn't target staging (vala), dropped vte, and removed gnome-shell color-picker patch.

Rebuilding.

**Comment by jtojnar** [comment]
gnome-shell and simple-scan can go to master too. And possibly also tracker-miners and gnome-desktop.

**Comment by fabianhjr** [comment]
@ofborg eval

**Pull request closed without merging**